bin/yaml/libraries.yaml

```diff
@@ -1070,17 +1070,17 @@ libraries:
         lib_type: static
         method: nightlyclone
         package_install: true
-        repo: beman-project/iterator_interface
+        repo: bemanproject/iterator_interface
         staticliblink:
         - beman.iterator_interface
         targets:
         - main
         type: github
-      beman_optional26:
+      beman_optional:
         build_type: none
         check_file: README.md
         method: nightlyclone
-        repo: beman-project/optional26
```

Comment:
As it stands this will cause current uses of optional to use the "old" version; this will create a new folder `/opt/compiler-explorer/libs/beman_optional` (leaving the old one alone). I'll merge this; then symlink the old to the new to keep it working; but a separate PR to the main CE to change the path would be great.

Reply:
@mattgodbolt , please also check https://github.com/compiler-explorer/compiler-explorer/pull/7328. Already pushed a PR in CE.

Reply:
Done:
```
$ ls -ld /opt/compiler-explorer/libs/beman*
drwxr-xr-x 3 root root 6144 Nov 12 00:19 /opt/compiler-explorer/libs/beman_iterator
drwxr-xr-x 3 root root 6144 Jan 11 00:20 /opt/compiler-explorer/libs/beman_iterator_interface
drwxr-xr-x 3 root root 6144 Jan 29 16:25 /opt/compiler-explorer/libs/beman_optional
lrwxrwxrwx 1 root root   14 Jan 29 16:26 /opt/compiler-explorer/libs/beman_optional26 -> beman_optional
```

Reply:
After the second PR is merged, I think you can remove the symlink.